src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -350,8 +350,9 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 		let exposureRiskMetadata = gatherExposureRiskMetadata()
 		let userMetadata = gatherUserMetadata()
 		let clientMetadata = gatherClientMetadata()
-		let pcrKeySubmissionMetadata = gatherKeySubmissionMetadata(for: .pcr)
-		let antigenKeySubmissionMetadata = gatherKeySubmissionMetadata(for: .antigen)
+		let pcrKeySubmissionMetadata = gatherKeySubmissionMetadata(for: .registeredTest(.pcr))
+		let antigenKeySubmissionMetadata = gatherKeySubmissionMetadata(for: .registeredTest(.antigen))
+		let srsKeySubmissionMetadata = gatherKeySubmissionMetadata(for: .srs(.srsOther))
```

Comment:
Any specific reason why we use .srsOther here?

Reply:
No buy the enum has an associated type and we cant intialize it without one, the actual type is passed into the metadata object already